packages/cli/src/constants.ts

```diff
@@ -41,3 +41,6 @@ export const UNKNOWN_FAILURE_REASON = 'Unknown failure reason';
 
 export const WORKFLOW_REACTIVATE_INITIAL_TIMEOUT = 1000;
 export const WORKFLOW_REACTIVATE_MAX_TIMEOUT = 180000;
+
+export const SETTINGS_LICENSE_CERT_KEY = 'license.cert';
+export const LICENSE_FEAT_SHARING_KEY = 'feat:sharing';
```

Comment:
Good idea to make these strings constants! It might even be worth to create an enum for them so that they are namespaced.

```typescript
enum LicenseFeature {
  SHARING = 'feat:sharing',
  FOO = 'feat:foo',
  BAR = 'feat:bar'
}

//...

import { getLicense } from '@/License';

if (getLicense().isFeatureEnabled(LicenseFeature.SHARING)) {
  //...
}
```

Reply:
👍🏽

Reply:
We should avoid raw enums:

- https://fettblog.eu/tidy-typescript-avoid-enums/
- https://www.executeprogram.com/blog/typescript-features-to-avoid 
- https://dev.to/ylerjen/use-typescript-const-assertion-instead-of-enums-mfn

Alternatives:

```ts
const enum MyConstEnum {
  A = 0,
  B = 1,
}

const myConstAssertionEnum = { A: 0, B: 1 } as const;
```

Reply:
ah, good point! A virtual enum does the trick nicely IMO, compiling just to the raw string values:

```typescript
export const enum LicenseFeature {
  SHARING = 'feat:sharing',
  FOO = 'feat:foo',
  BAR = 'feat:bar',
}

//...

import { getLicense } from '@/License';

if (getLicense().isFeatureEnabled(LicenseFeature.SHARING)) {
  //...
}
```